.github/workflows/unit_test_main.yaml

```diff
@@ -59,3 +59,18 @@ jobs:
         working-directory: tests
         run: |
           python3 -m pytest ./
+
+      - name: Generate coverage report
+        run: |
+          rm -rf ./coverage.xml
+          pip install coverage codecov pytest pytest-cov
+          coverage erase
+          coverage run --source=gptcache  -m pytest ./tests/unit_tests
```

Comment:
should include the integration, like `-m pytest ./tests`

Reply:
OK